frontend/src/components/chat/ChatInterface.tsx

```diff
@@ -43,6 +49,27 @@ function ChatInterface() {
   const { messages } = useSelector((state: RootState) => state.chat);
   const { curAgentState } = useSelector((state: RootState) => state.agent);
 
+  const [feedbackShared, setFeedbackShared] = React.useState(false);
+
+  const shareFeedback = async (feedback: "positive" | "negative") => {
+    const data: FeedbackData = {
+      email: "NOT_PROVIDED",
+      token: getToken(),
+      feedback,
+      trajectory: removeApiKey(Session._history),
+    };
+
+    try {
+      await sendFeedback(data);
```

Comment:
Here, could you please check that the result of the request has status code 200, and if not report the error to the console?

Reply:
@neubig Because I had to set the mode to `'no-cors'`, I do not get any response that I can use. Anyways, the `try..catch` block that it is wrapped around acts like the `try...except` in Python. If the request fails, it logs the error to console and displays an error toast pop up.

Reply:
Actually, can you confirm that you receive the data in your backend? I have already sent a few.